Packages/com.unity.render-pipelines.universal/Runtime/RenderingUtils.cs

```diff
@@ -1116,10 +1116,10 @@ static public DrawingSettings CreateDrawingSettings(List<ShaderTagId> shaderTagI
         /// <param name="destination"></param>
         /// <param name="cameraData"></param>
         /// <returns></returns>
-        internal static Vector4 GetFinalBlitScaleBias(RTHandle source, RTHandle destination, UniversalCameraData cameraData)
+        internal static Vector4 GetFinalBlitScaleBias(RTHandle source, RTHandle destination, UniversalCameraData cameraData, bool nrpRenderGraph=false)
         {
             Vector2 viewportScale = source.useScaling ? new Vector2(source.rtHandleProperties.rtHandleScale.x, source.rtHandleProperties.rtHandleScale.y) : Vector2.one;
-            var yflip = cameraData.IsRenderTargetProjectionMatrixFlipped(destination);
+            var yflip = cameraData.IsRenderTargetProjectionMatrixFlipped(destination) || (SystemInfo.graphicsDeviceType == GraphicsDeviceType.Vulkan && nrpRenderGraph);
```

Comment:
How is this not already handled with IsRenderTargetProjectionMatrixFlipped? If we really need this, I'd argue that we should check for `graphicsUVStartsAtTop` instead of API == Vulkan

Reply:
IsRenderTargetProjectionMatrixFlipped think the final destination is not yflipped, which is correct. The issue here is by default C# code think the intermediate texture is up-side down, but this is not the case as we make sure all the intermediate texture is rendered up-side up with other subpass changes.